lmctl/client/models/intents.py

```diff
@@ -94,6 +94,33 @@ def __eq__(self, other):
         return (isinstance(other, DeleteAssemblyIntent) and self.assembly_name==other.assembly_name
             and self.assembly_id==other.assembly_id)
 
+class RetryAssemblyIntent(ExistingAssemblyIntent):
```

Comment:
These requests expect a `process_id`, not a `assembly_id` so should use that name in the constructors. This also means they shouldn't inherit from `ExistingAssemblyIntent`

Reply:
Addressed